config/robots.rb

```diff
@@ -3061,4 +3061,40 @@
         }
       }
     )
+
+    # LRC Hamilton Star bed verification
+    # LRC PBMC Bank to LRC Bank Seq and LRC Bank Spare
+    custom_robot(
+      'hamilton-lrc-pbmc-bank-to-lrc-bank-seq-and-lrc-bank-spare',
+      name: 'Hamilton LRC PBMC Bank => LRC Bank Seq and LRC Bank Spare',
+      beds: {
+        bed(12).barcode => {
+          purpose: 'LRC PBMC Bank',
+          states: ['passed'],
+          label: 'Bed 12'
+        },
+        bed(15).barcode => {
+          purpose: 'LRC Bank Seq',
+          states: ['pending'],
+          label: 'Bed 15',
+          target_state: 'passed'
+        },
+        bed(14).barcode => {
+          purpose: 'LRC Bank Spare',
+          states: ['pending'],
+          label: 'Bed 14',
+          target_state: 'passed'
+        }
+      },
+      class: 'Robots::PlateToTubeRacksRobot',
+      relationships: [
+        {
+          'type' => 'LRC PBMC Bank to LRC Bank Seq/Spare',
```

Comment:
The relationships section in robot config is an array. It appears whenever there is a relationships section in the robots configuration file, i.e., 'Bravo PF-384 Lib XP2 => PF-Lib Q-XP2' (as 'quad_stamp_out') and 'Bravo CLCM Stock => CLCM Lysate DNA and RNA' (as 'RNA and DNA split' robot configurations. I guess, initially it was designed to accept multiple relationships that can be distinguished by 'type'.  Codes for handing relationships use a loop because of this structure. I followed the convention and added some 'type' information. However, it is not used by this robot and it can be removed.